deployment/kubernetes/charts/origin/templates/dapp.ingress.yaml

```diff
@@ -12,6 +12,7 @@ metadata:
     kubernetes.io/tls-acme: "true"
     certmanager.k8s.io/cluster-issuer: {{ .Values.clusterIssuer }}
     nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
+    nginx.ingress.kubernetes.io/limit-rps: "2"
```

Comment:
Does that include serving static content ? If yes, that seems low. When I reload the Dapp in my browser, I see in the order of 30 requests to get static content (css, images).